elasticsearch/sgconfig/sg_roles_mapping.yml

```diff
@@ -22,5 +22,8 @@ sg_role_prometheus:
     - 'prometheus'
 
 sg_role_jaeger:
+  users:
+    - "CN=jaeger.user,OU=OpenShift,O=Logging"
```

Comment:
so this is a client cert auth user instead of a token user?

Reply:
Where is this client cert/key created?  How does Elasticsearch/SG trust the CA cert of the CA that issued the client cert?

Reply:
The certs are created in jaeger-operator using this script: https://github.com/jaegertracing/jaeger-operator/pull/191/files#diff-5a317832296dcbda4dee718589dca819R3

I have followed the same approach as in logging-operator.

Reply:
lgtm - any further objections @ewolinetz or @jcantrill ?

Reply:
doesn't this make it so it isn't using the token but rather certs? if we're doing that I don't think we need the change in sg_config.yml

Reply:
It is correct that if we define the CN from the certificate there should be no need to modify 'sg_config' . I would recommend one or the other

Reply:
I left the cert options here as a backup solution. We are using token at the moment.

Reply:
ok I have removed user certs for now as not needed